### ISSUE TITLE
ModuleNotFoundError: No module named 'pymumble.pymumble_py3'

### ISSUE THREAD

**Issue by ThomasKoscheck** [DuckBoss/JJMumbleBot#102]
When trying to start the bot with `python JJMumbleBot` i got the following error:

```
Traceback (most recent call last):
   File "/usr/local/lib/python3.7/runpy.py", line 193, in _run_module_as_main
     "__main__", mod_spec)
   File "/usr/local/lib/python3.7/runpy.py", line 85, in _run_code
     exec(code, run_globals)
  File "./JJMumbleBot/__main__.py", line 1, in <module>
    import JJMumbleBot
  File "./JJMumbleBot/JJMumbleBot.py", line 1, in <module>
     import pymumble.pymumble_py3 as pymumble
ModuleNotFoundError: No module named 'pymumble.pymumble_py3'
```
Do you have an idea where this error comes from?
Thank you!

**Comment by DuckBoss** [comment]
This happens when the pymumble library folder isn't located in the root directory of the bot.
Make sure that your pymumble folder is named "pymumble" and it contains the "pymumble_py3" folder.
This bot is also only supported for Python 3.6+ which could potentially cause an importing error.

The general file structure should look like this:
```
> JJMumbleBot
    > ...
    > ...
    > pymumble
        > pymumble_py3
    > ...
    > ...
```

**Comment by ThomasKoscheck** [comment]
Thank you for your hint. After cloning pymumble manually into tje JJMumbleBot folder this error disappears.
I also downgraded from python 3.7 to 2.7 to prevent error from the wrong python version.

But now, this error occurs. 
```
Traceback (most recent call last):
   File "/usr/local/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
   File "/usr/local/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
   File "/usr/src/app/JJMumbleBot/__main__.py", line 1, in <module>
     import JJMumbleBot
   File "./JJMumbleBot/JJMumbleBot.py", line 48
     log_file_name = f"{GM.cfg['Bot_Directories']['LogDirectory']}/runtime.log"
 SyntaxError: invalid syntax
```

**Comment by DuckBoss** [comment]
The minimum python version is 3.6, so python 2.7 wouldn't work for this bot unfortunately. It's giving you that syntax error because the "f-string formatting" wasn't implemented before python 3.6

**Comment by DuckBoss** [comment]
I'm closing this issue if there are no further questions/comments.
